### PR TITLE
inardini - experiments cuj1 notebook release

### PR DESCRIPTION
**Pull request by inardini**
A PR to release the official experiments notebook - CUJ1.

If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [x] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/experiments/comparing_local_trained_models.ipynb [hunk 1]
```diff
@@ -0,0 +1,1070 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:1]: <p>Suggest under what you learn to include:</p><p> </p><p>Comparing two experiments</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/636/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:1]: <p>steps(for  -&gt; steps (for</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/636/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Comment (inardini) [reply notebooks/official/experiments/comparing_local_trained_models.ipynb:1]: <p>done</p>

Comment (inardini) [reply notebooks/official/experiments/comparing_local_trained_models.ipynb:1]: <p>done</p>

**Review by andrewferlitsch — APPROVED** [review]

**Comment by andrewferlitsch** [comment]
test failed on unrelated notebook: notebook_template.ipynb

----> 5 import tensorflow as tf
Step #3:       7 model = tf.keras.Sequential(
Step #3:       8     [
Step #3:       9         tf.keras.layers.Dense(10, activation="relu", input_shape=(None, 5)),
Step #3:      10         tf.keras.layers.Dense(3),
Step #3:      11     ]
Step #3:      12 )
Step #3: 
Step #3: File /usr/local/lib/python3.9/site-packages/tensorflow/__init__.py:37, in <module>
Step #3:      34 import sys as _sys
Step #3:      35 import typing as _typing
Step #3: ---> 37 from tensorflow.python.tools import module_util as _module_util
Step #3:      38 from tensorflow.python.util.lazy_loader import LazyLoader as _LazyLoader
Step #3:      40 # Make sure code inside the TensorFlow codebase can use tf2.enabled() at import.
Step #3: 
Step #3: File /usr/local/lib/python3.9/site-packages/tensorflow/python/__init__.py:42, in <module>
Step #3:      37 from tensorflow.python.eager import context
Step #3:      39 # pylint: enable=wildcard-import
Step #3:      40 
Step #3:      41 # Bring in subpackages.
Step #3: ---> 42 from tensorflow.python import data
Step #3:      43 from tensorflow.python import distribute
Step #3:      44 # from tensorflow.python import keras
Step #3: 
Step #3: File /usr/local/lib/python3.9/site-packages/tensorflow/python/data/__init__.py:21, in <module>
Step #3:      15 """`tf.data.Dataset` API for input pipelines.
Step #3:      16 
Step #3:      17 See [Importing Data](https://tensorflow.org/guide/data) for an overview.
Step #3:      18 """
Step #3:      20 # pylint: disable=unused-import
Step #3: ---> 21 from tensorflow.python.data import experimental
Step #3:      22 from tensorflow.python.data.ops.dataset_ops import AUTOTUNE
Step #3:      23 from tensorflow.python.data.ops.dataset_ops import Dataset
Step #3: 
Step #3: File /usr/local/lib/python3.9/site-packages/tensorflow/python/data/experimental/__init__.py:95, in <module>
Step #3:      15 """Experimental API for building input pipelines.
Step #3:      16 
Step #3:      17 This module contains experimental `Dataset` sources and transformations that can
Step #3:    (...)
Step #3:      91 @@UNKNOWN_CARDINALITY
Step #3:      92 """
Step #3:      94 # pylint: disable=unused-import
Step #3: ---> 95 from tensorflow.python.data.experimental import service
Step #3:      96 from tensorflow.python.data.experimental.ops.batching import dense_to_ragged_batch
Step #3:      97 from tensorflow.python.data.experimental.ops.batching import dense_to_sparse_batch
Step #3: 
Step #3: File /usr/local/lib/python3.9/site-packages/tensorflow/python/data/experimental/service/__init__.py:387, in <module>
Step #3:       1 # Copyright 2020 The TensorFlow Authors. All Rights Reserved.
Step #3:       2 #
Step #3:       3 # Licensed under the Apache License, Version 2.0 (the "License");
Step #3:    (...)
Step #3:      13 # limitations under the License.
Step #3:      14 # ==============================================================================
Step #3:      15 """API for using the tf.data service.
Step #3:      16 
Step #3:      17 This module contains:
Step #3:    (...)
Step #3:     384   job of ParameterServerStrategy).
Step #3:     385 """
Step #3: --> 387 from tensorflow.python.data.experimental.ops.data_service_ops import distribute
Step #3:     388 from tensorflow.python.data.experimental.ops.data_service_ops import from_dataset_id
Step #3:     389 from tensorflow.python.data.experimental.ops.data_service_ops import register_dataset
Step #3: 
Step #3: File /usr/local/lib/python3.9/site-packages/tensorflow/python/data/experimental/ops/data_service_ops.py:22, in <module>
Step #3:      20 from tensorflow.core.protobuf import data_service_pb2
Step #3:      21 from tensorflow.python import tf2
Step #3: ---> 22 from tensorflow.python.data.experimental.ops import compression_ops
Step #3:      23 from tensorflow.python.data.experimental.service import _pywrap_server_lib
Step #3:      24 from tensorflow.python.data.experimental.service import _pywrap_utils
Step #3: 
Step #3: File /usr/local/lib/python3.9/site-packages/tensorflow/python/data/experimental/ops/compression_ops.py:16, in <module>
Step #3:       1 # Copyright 2020 The TensorFlow Authors. All Rights Reserved.
Step #3:       2 #
Step #3:       3 # Licensed under the Apache License, Version 2.0 (the "License");
Step #3:    (...)
Step #3:      13 # limitations under the License.
Step #3:      14 # ==============================================================================
Step #3:      15 """Ops for compressing and uncompressing dataset elements."""
Step #3: ---> 16 from tensorflow.python.data.util import structure
Step #3:      17 from tensorflow.python.ops import gen_experimental_dataset_ops as ged_ops
Step #3:      20 def compress(element):
Step #3: 
Step #3: File /usr/local/lib/python3.9/site-packages/tensorflow/python/data/util/structure.py:22, in <module>
Step #3:      19 import six
Step #3:      20 import wrapt
Step #3: ---> 22 from tensorflow.python.data.util import nest
Step #3:      23 from tensorflow.python.framework import composite_tensor
Step #3:      24 from tensorflow.python.framework import ops
Step #3: 
Step #3: File /usr/local/lib/python3.9/site-packages/tensorflow/python/data/util/nest.py:36, in <module>
Step #3:      16 """## Functions for working with arbitrarily nested sequences of elements.
Step #3:      17 
Step #3:      18 NOTE(mrry): This fork of the `tensorflow.python.util.nest` module
Step #3:    (...)
Step #3:      31    arrays.
Step #3:      32 """
Step #3:      34 import six as _six
Step #3: ---> 36 from tensorflow.python.framework import sparse_tensor as _sparse_tensor
Step #3:      37 from tensorflow.python.util import _pywrap_utils
Step #3:      38 from tensorflow.python.util import nest
Step #3: 
Step #3: File /usr/local/lib/python3.9/site-packages/tensorflow/python/framework/sparse_tensor.py:24, in <module>
Step #3:      22 from tensorflow.python import tf2
Step #3:      23 from tensorflow.python.framework import composite_tensor
Step #3: ---> 24 from tensorflow.python.framework import constant_op
Step #3:      25 from tensorflow.python.framework import dtypes
Step #3:      26 from tensorflow.python.framework import ops
Step #3: 
Step #3: File /usr/local/lib/python3.9/site-packages/tensorflow/python/framework/constant_op.py:25, in <module>
Step #3:      23 from tensorflow.core.framework import types_pb2
Step #3:      24 from tensorflow.python.eager import context
Step #3: ---> 25 from tensorflow.python.eager import execute
Step #3:      26 from tensorflow.python.framework import dtypes
Step #3:      27 from tensorflow.python.framework import op_callbacks
Step #3: 
Step #3: File /usr/local/lib/python3.9/site-packages/tensorflow/python/eager/execute.py:19, in <module>
Step #3:      15 """Functions called by the generated code to execute an eager-mode op."""
Step #3:      17 import six
Step #3: ---> 19 from google.protobuf import text_format
Step #3:      20 from tensorflow.core.framework import tensor_pb2
Step #3:      21 from tensorflow.python import pywrap_tfe
Step #3: 
Step #3: ImportError: cannot import name 'text_format' from 'google.protobuf' (/builder/home/.local/lib/python3.9/site-packages/google/protobuf/__init__.py)
Step #3: 
Finished Step #3

**Review by ivanmkc — APPROVED** [review]
LGTM

Execution test passed (though it keeps running in Github due to a bug in Cloud Build).

Please make sure the links at the top work after you merge.

**Comment by inardini** [comment]
@ivanmkc ready to merge

**Pull request merged**